streams/src/test/java/org/apache/kafka/streams/integration/RestoreIntegrationTest.java

```diff
@@ -97,11 +102,13 @@ public class RestoreIntegrationTest {
 
     public static final EmbeddedKafkaCluster CLUSTER = new EmbeddedKafkaCluster(NUM_BROKERS);
 
+
```

Comment:
Are these new lines intentional? If not could you revert them?

Reply:
Used a formatter that seems to have added a couple of few lines. Removed 👍